src/sandbox/utils/resolve-module.js

```diff
@@ -3,7 +3,7 @@ import type { Module, Directory } from 'common/types';
 
 const compareTitle = (original: string, test: string) => {
   if (original === test) return true;
-  if (original === `${test}.js`) return true;
+  if (original.match(new RegExp(`^${test}.jsx?$`))) return true;
```

Comment:
Hmm, I'd prefer to let this still be a strict equal, since the case is very clear and building Regexes can be quite expensive.